### DISCUSSION TITLE
Integration mit Solaranzeige?

### DISCUSSION THREAD

**Discussion by hans-dot** [tbnobody/OpenDTU#426] [Ideas]
Hallo openDTU Entwickler: 
auf solaranzeige.de gibts ein Raspi Image das eine Menge kann. Habe openDTU laufen und auch das Raspi mit solaranzeige:
Die Daten werden dort von masquitto gelesen:
mosquitto_sub -t "#" -v 
solar/112171909861/1/voltage 31.80
...usw
Kann jemand die Integration mit Solaranzeige einbringen.
Vielen Dank!

**Comment by MrReSc** [comment]
Ich denke die Integration muss von Seitens solaranzeige.de kommen. Die Wechselrichter senden ja nur die Daten und solaranzeige.de bereitet sie auf. Das selbe gilt auch für OpenDTU.
